templates/activity.mustache

```diff
@@ -59,6 +59,12 @@
     {{#templatelist}}
         var templateList = {{ templatelist }};
     {{/templatelist}}
+    {{#activityid}}
```

Comment:
Should this be here?  I don't think it's part of this feature...

Reply:
My mistake it crept in from another branch, sorry!